src/utils/player.ts

```diff
@@ -162,8 +162,9 @@ class Player {
       } else return null;
     }
     // 返回歌曲地址
-    // 客户端直接返回，网页端转 https
-    const url = isElectron ? songData.url : songData.url.replace(/^http:/, "https:");
+    // 客户端直接返回，网页端转 https, 并转换url以便解决音乐链接cors问题
+    const url = isElectron ? songData.url : songData.url.replace(/^http:/, "https:").replace(/m804\.music\.126\.net/g, 'm801.music.126.net').replace(/m704\.music\.126\.net/g, 'm701.music.126.net');
+    console.log(`🎧 ${id} music url:`, url);
```

Comment:
在生产环境中，应避免打印敏感信息（如完整的音乐URL）到控制台，以防止潜在的安全风险。

```suggestion
// 可以考虑仅在开发模式下打印详细日志
    if (process.env.NODE_ENV === 'development') {
      console.log(`🎧 ${id} music url:`, url);
    }